docs/contracts.rst

```diff
@@ -22,43 +22,67 @@ To run this example, you will need to install a few extra features:
 .. code-block:: python
```

Comment:
This whole code block can be removed since we'll be testing it starting at the `.. doctest::` line